1-js/05-data-types/02-number/article.md

```diff
@@ -331,35 +331,35 @@ There is a special built-in method [Object.is](mdn:js/Object/is) that compares v
 
 In all other cases, `Object.is(a, b)` is the same as `a === b`. 
 
-This way of comparison is often used in JavaScript specification. When an internal algorithm needs to compare two values for being exactly the same, it uses `Object.is` (internally called [SameValue](https://tc39.github.io/ecma262/#sec-samevalue)).
+This method of comparison is often used in JavaScript. When an internal algorithm needs to compare two values for being exactly the same, it uses `Object.is` (internally called [SameValue](https://tc39.github.io/ecma262/#sec-samevalue)).
 ```
 
 
 ## parseInt and parseFloat
 
-The numeric conversion using a plus `+` or `Number()` is strict. If a value is not exactly a number, it fails:
+Numeric conversion using a plus `+` or `Number()` is strict. If a value is not exactly a number, it fails:
 
 ```js run
 alert( +"100px" ); // NaN
 ```
 
-The sole exception is spaces before and after the line, they are ignored.
+The sole exception is spaces before and after the line, as they are ignored.
```

Comment:
Maybe: "The sole exception is spaces at the beginning or at the end of the string..." ?

Reply:
@iliakan that would definitely be more specific